shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -710,6 +710,11 @@ - (void)forwardInvocation:(NSInvocation*)anInvocation {
 
 @end
 
+@interface FlutterTextInputPlugin ()
+- (id<FlutterTextInputDelegate>)textInputDelegate;
+- (fml::WeakPtr<FlutterTextInputPlugin>)getWeakPtr;
```

Comment:
"getWeakPtr" isn't idiomatic, you should just use a readonly property for this called `weakPtr`.